components/ResultViewList.tsx

```diff
@@ -31,7 +31,12 @@ const ResultViewList = ({
           {displayName && (
             <span
               className="sui-result__title"
-              dangerouslySetInnerHTML={{ __html: displayName }}
+              dangerouslySetInnerHTML={{
+                __html: displayName.replace(
+                  /<em[^>]*>/g,
+                  '<em style="background-color: transparent;">'
+                ),
+              }}
```

Comment:
Could I check what the purpose of this code is?

Reply:
> I used `dangerouslySetInnerHTML= {{  __html: displayName.replace( /<em[^>]*>/g, '<em style="background-color: transparent;">'), }}` to remove the background colour styling for the search result in the expanded card view as there were 2 overlapping background colours in the expanded card view. 
> 
> 
> 
> <img width="251" alt="Screenshot 2023-10-21 at 3 48 18 PM" src="https://github.com/AdvisorySG/mentorship-page/assets/73103636/06c2e362-390d-4650-b731-5de46f732305">
> 
> 

I added this code to prevent the double highlighting.

Reply:
Ah okay, makes sense. Thanks!